doc/changes/latest.inc

```diff
@@ -49,6 +49,8 @@ Bugs
 
 - Fix bug with :func:`mne.io.read_raw_nicolet` where header type values such as num_sample and duration_in_sec where not parsed properly (:gh:`8712` by `Alex Gramfort`_)
 
+- Fix bug with :func:`mne.preprocessing.read_ica_eeglab` with reading AMICA decompositions (:gh:`8780` by `Alex Gramfort`_ and `Eric Larson`_)
```

Comment:
I don't think this is a problem specific to AMICA; all ICA decompositions with principal components removed should be affected.